components/com_contact/helpers/association.php

```diff
@@ -10,6 +10,7 @@
 defined('_JEXEC') or die;
```

Comment:
this PR must not have changes to thios helpers.
They are already on gsoc staging. UPDATE your branches and this will disappear.